components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -86,13 +83,6 @@ class PageWallet extends React.Component<Props, State> {
     })
   }
 
-  onModalBackupOnRestore = (key: string | MouseEvent) => {
-    if (typeof key === 'string' && key.length > 0) {
-      key = this.pullRecoveryKeyFromFile(key)
```

Comment:
I think we can now remove `pullRecoveryKeyFromFile`, too.

Reply:
Thanks, removed!